packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig

```diff
@@ -27,21 +33,25 @@
   {% else %}
     </header>
   {% endif %}
-  <div class="ma__action-step__content {{ accordion ? 'js-accordion-content' : '' }}">
-  {% set richText = actionStep.richText %}
-  {% include "@organisms/by-author/rich-text.twig" %}
-  {% block formDownloads %}
-    {% if actionStep.downloadLinks %}
-      {% set formDownloads = { "downloadLinks": actionStep.downloadLinks } %}
-      {% include "@organisms/by-author/form-downloads.twig" %}
-      {% endif %}
-  {% endblock %}
-  {% block decorativeLink %}
-  {% if actionStep.decorativeLink %}
-    <div class="ma__action-step__more">
-      {% set decorativeLink = actionStep.decorativeLink %}
-      {% include "@atoms/decorative-link.twig" %}
-    </div>
+  {% if actionStep.richText or actionStep.decorativeLink %}
+    <div class="ma__action-step__content {{ accordion ? 'js-accordion-content' : '' }}">
+    {% if actionStep.richText %}
+      {% set richText = actionStep.richText %}
+      {% include "@organisms/by-author/rich-text.twig" %}
+    {% endif %}
+    {% block formDownloads %}
+      {% if actionStep.downloadLinks %}
+        {% set formDownloads = { "downloadLinks": actionStep.downloadLinks } %}
+        {% include "@organisms/by-author/form-downloads.twig" %}
+        {% endif %}
+    {% endblock %}
+    {% block decorativeLink %}
+    {% if actionStep.decorativeLink %}

```

Comment:
I don't think you need to add a separate decorativeLink component here, you can just pass the decorative component as part of the rich Text data object.

Reply:
Instead of adding decorativeLink into the template like this, you can just pass a decorative link into the richText data object like this:
```
    {
      "accordion": true,
      "isExpanded": false,
      "accordionLabel": "Expand DUA warns about phising scam",
      "icon": "input-error",
      "icon_label": "alert",
      "level": "3",
      "title": "DUA warns about phising scam",
      "suffix": "Updated: Mar. 23rd, 2021, 9:50 am",
      "richText": {
        "rteElements": [{
          "path": "@atoms/decorative-link.twig",
          "data": {
            "decorativeLink": {
              "href": "#",
              "text": "Lorem ipsum dolor sit amet.",
              "context": "",
              "button": false,
              "info": "",
              "labelContext": "",
              "ariaLabel": ""
            }
          }
        }]
      },
      "decorativeLink": {
        "href": "#",
        "text": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus deleniti nisi at adipisci sequi consequuntur qui, corporis pariatur eos laborum nostrum itaque, facere. Doloremque qui temporibus repellendus incidunt in nam provident, aliquam.",
        "info": "Learn more about DUA warns about phising scam",
        "property": ""
      }
    },
```

![Screen Shot 2021-07-26 at 1 15 08 PM](https://user-images.githubusercontent.com/5789411/127031179-d1bc2e47-7da9-4ac9-876d-16c133098eea.png)

Reply:
nevermind about this.. It was not added by PR. Not sure why it was added separately, to begin with. Ignore this comment